hbase-client/src/main/java/org/apache/hadoop/hbase/client/BufferedMutator.java

```diff
@@ -179,23 +179,35 @@ default long getWriteBufferPeriodicFlushTimerTickMs() {
    * The default value comes from the configuration parameter {@code hbase.client.write.buffer}.
    * @return The size of the write buffer in bytes.
    */
-  long getWriteBufferSize();
+  default long getWriteBufferSize() {
+    throw new UnsupportedOperationException(
+      "The BufferedMutator::getWriteBufferSize has not been implemented"
+    );
+  }
 
   /**
    * Set rpc timeout for this mutator instance
    * @deprecated Since 3.0.0, will be removed in 4.0.0. Please set this through the
    *             {@link BufferedMutatorParams}.
    */
   @Deprecated
-  void setRpcTimeout(int timeout);
+  default void setRpcTimeout(int timeout) {
```

Comment:
this will also help downstream users prepare for the removal in 4.0.0 by removing any overloads they might have added in delegate classes, etc.